### DISCUSSION TITLE
Switch entry’s site

### DISCUSSION THREAD

**Discussion by bertheyman** [craftcms/cms#8824] [Ideas]
Is it possible to switch the subsite of an existing entry?
If not, is there any chance this might be added in the future?

**Comment by brandonkelly** [comment]
What do you mean by “subsite”?

**Comment by bertheyman** [comment]
Hi @brandonkelly , and sorry for the late reply!

An example: a shop owner has three shops. He manages them from one Craft installation with three sites in it via the multi-site feature.

News items are specific per shop, so propagate entries across sites is off.
Every new article will only be visible for the relevant shop, which is exactly working great for them.
But in some cases, articles are shared: would it be possible to duplicate the article and change the owning site (shop)?

**Comment by brandonkelly** [comment]
Gotcha… that’s not currently possible via the UI. I’ll mark this as a feature request.
